netsocadmin/register_tools.py

```diff
@@ -23,6 +23,50 @@
 ldap_server = ldap3.Server(config.LDAP_HOST, get_info=ldap3.ALL)
 
 
+def send_forgot_email(email: str, server_url: str) -> bool:
+    """
+    Sends email containing the user's username and the link which users use to
+    generate a new password
+
+    :param email the email address which the user registered with
+    :param server_url the address of the flask application
+    :returns boolean true if the email was sent succesfully, false otherwise.
+    """
+
+    conn = pymysql.connect(**config.MYSQL_DETAILS)
+    user = ""
+    with conn.cursor() as c:
+        sql = "SELECT uid FROM users WHERE email=%s;"
+        c.execute(sql, (email,))
+        user = c.fetchone()[0]
+
+    uri = generate_uri(email)
+    message_body = f"""
+Hello,
+
+Your server log-in is as follows:
+
+username: {user}
+
+If you wish to reset your password then click the link below.
+
+http://{server_url}/resetpassword?t={uri}&e={email}&u={user}
+
+Yours,
+
+The UCC Netsoc SysAdmin Team
+"""
+    if not config.FLASK_CONFIG['debug']:
+        response = mail_helper.send_mail(
+            "server.registration@netsoc.co",
+            email,
+            "Account Details",
```

Comment:
Same as the other point https://github.com/UCCNetsoc/netsocadmin2/pull/72/files#diff-72835049171ebe8f6353d7064e9126cbR363